access-control-app/src/cmr/access_control/system.clj

```diff
@@ -31,26 +31,52 @@
   {:default nil
    :parser cfg/maybe-long})
 
+(defconfig log-level
+  "App logging level"
+  {:default "info"})
+
 (defconfig access-control-public-protocol
-  "The protocol to use in documentation examples for the access-control application."
+  "The protocol to use in documentation examples for the access-control application.
+
+  Note: this configuration value is used as-is in local, dev environments
+  and is overridden with ENV variables in remote deployments. In both cases,
+  this configuration information is utilized and required.
+
+  The name given after `defconfig` is important and is used when checking for
```

Comment:
The documentation on the variable name is probably redundant with the defconfig macro itself. Rather than listing what the environment variable name will be in each place, we should probably make sure the defconfig macro documentation is thorough and includes the environment variable naming.